templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -32,20 +32,33 @@
  #}
 
 {% set timeline_display_date = session('glpitimeline_date_format') %}
+{% set date_creation_formatted = date_creation %}
+{% set date_mod_formatted = date_mod %}
+{% set date_creation_relative = date_creation %}
+{% set date_mod_relative = date_mod %}
+
+{% if not (anchor starts with 'Log_') %}
+    {# Workaround issue where log data already has a formatted datetime #}
```

Comment:
```suggestion
    {# Workaround issue where log data already has a formatted datetime -- see https://github.com/glpi-project/glpi/pull/15901 #}
```

Also a comment for 10.1so we can drop formatting on Log would be a good thing I guess.

OK for me appart of that.

Reply:
```suggestion
    {# Workaround issue where log data already has a formatted datetime -- see https://github.com/glpi-project/glpi/pull/15901 #}
    {# TODO in GLPI 10.1: Remove this workaround #}
```